Prog/clean-Loudoun.Rmd

```diff
@@ -0,0 +1,1005 @@
+---
+title: "Regional Housing Framework"
+author: ''
+subtitle: Clean Loudoun County public records data
+output:
+  pdf_document:
```

Comment:
@KScott-Urban some of your errors are because you are knitting to a PDF here. if you look clean-arlington.Rmd the output is `html_document` . Is there a reason you are knitting to a PDF?

Reply:
Hi Sarah, no particular reason. Just made the wrong assumption. Thanks for letting me know. I will try to knit an html doc.

Reply:
The html docs are now ready! Thank you, Sarah.

Reply:
thanks! can you commit and push your changes?

Reply:
Now committed and pushed.